src/mono/wasm/debugger/tests/debugger-test/debugger-test.cs

```diff
@@ -1371,4 +1371,17 @@ public static void CheckArguments(ReadOnlySpan<object> parameters)
     {
         System.Diagnostics.Debugger.Break();
     }
+}
+
+public class ToStringOverwritten
+{
+    public override string ToString()
+    {
+        return "hello";
+    }
+    public static void Run()
```

Comment:
Can we also add a tests for:

```csharp
class A { public override string ToString() { ... } }
class B: A { .. }

var b = new B();
```
and
```csharp
class A { .. }
class B: A { public override string ToString() { .. } }

A a = new B();
```

Reply:
Along with `A a = new B();`, `object o = new B();` would be useful too.

Reply:
done